### PR TITLE
[Tests] Add checkNumericalEquivalence() calls to a couple of GraphOptz tests

### PR DESCRIPTION
**Pull request by soumyadeepghosh**
Summary:
Included additional checks for numerical equivalence of base and optimized functions for the
following two tests in `GraphOptzTest`:
1. `batchNormAfterConvNotOptimizeWhenMoreThanOneUseOfConv`
2. `foldQuantizeIntoVarMultipleUsages`

Test Plan: `ninja test`

### CODE REVIEW

**Comment by SplitInfinity** [comment]
Looks good! Can you attach the original and optimized dot graphs just as a sanity check to make sure the graphs being tested are different? I think this applies only to `foldQuantizeIntoVarMultipleUsages` because `batchNormAfterConvNotOptimizeWhenMoreThanOneUseOfConv` tests that an optimization does *not* run.

**Comment by soumyadeepghosh** [comment]
Attached are the base and the optimized graphs for `foldQuantizeIntoVarMultipleUsages`

[base.pdf](https://github.com/pytorch/glow/files/4100271/base.pdf)
[opti.pdf](https://github.com/pytorch/glow/files/4100272/opti.pdf)

**Comment by SplitInfinity** [comment]
@soumyadeepghosh Do you plan to land this?
